src/components/SettingsDrawer.jsx

```diff
@@ -0,0 +1,194 @@
+            </Select>
+          </ListItem>
+          <Divider />
+          <ListItem button sx={{ marginBottom: "8px" }}>
+            <ListItemIcon>
+              <SettingsBrightnessIcon />
+            </ListItemIcon>
+            <ListItemText primary="Theme Mode" />
+          </ListItem>
+          <Box sx={{ display: "flex", gap: "8px", marginTop: "8px", marginLeft: "48px" }}>
+            <Button
+              variant={themeMode === "light" ? "contained" : "outlined"}
+              onClick={() => handleThemeChange("light")}
+              startIcon={<Brightness7Icon />}
+            >
+              Light
+            </Button>
+            <Button
+              variant={themeMode === "system" ? "contained" : "outlined"}
+              onClick={() => handleThemeChange("system")}
+              startIcon={<SettingsBrightnessIcon />}
+            >
+              System
+            </Button>
+            <Button
+              variant={themeMode === "dark" ? "contained" : "outlined"}
+              onClick={() => handleThemeChange("dark")}
+              startIcon={<Brightness4Icon />}
+            >
+              Dark
+            </Button>
+          </Box>
+          <Divider />
+          <ListItem sx={{ marginBottom: "8px" }}>
+            <ListItemIcon>
+              <PaletteIcon sx={{ color: "gray" }} />
+            </ListItemIcon>
+            <ListItemText primary="Theme Color" />
+          </ListItem>
+          <Box sx={{ display: "flex", justifyContent: "center", marginTop: "8px", gap: "8px", marginLeft: "48px" }}>
+            {themeColors.map((color) => (
+              <Button
+                key={color}
+                variant={themeColor === color ? "contained" : "outlined"}
+                style={{ marginRight: "8px" }}
+                onClick={() => handleColorChange(color)}
+              >
+                <Typography>{color}</Typography>
+              </Button>
+            ))}
+          </Box>
+          <Divider />
+          <ListItem sx={{ marginBottom: "8px", display: "flex", alignItems: "center" }}>
+            <ListItemIcon>
+              <BusinessIcon />
+            </ListItemIcon>
+            <ListItemText primary="Tenant" />
+            <Select value={selectedTenant} onChange={handleTenantChange} size="small" sx={{ marginLeft: "auto" }}>
+              {tenantList.map((tenant) => (
+                <MenuItem key={tenant.id} value={tenant.id}>
+                  {tenant.name}
+                </MenuItem>
+              ))}
+            </Select>
+          </ListItem>
+        </List>
+        <Divider />
+        <div style={{ marginTop: "16px" }}>
+          {isAuthenticated ? (
+            <Button onClick={handleLogout} fullWidth variant="contained" color="primary">
+              Logout
+            </Button>
+          ) : (
+            <Button onClick={handleLogin} fullWidth variant="contained" color="primary">
+              Login
+            </Button>
+          )}
```

Comment:
Button is same just the text label & function is different for the button , we can simplify it minimising the code and also please use styled for css overrides or styles as we are doing it in other components